applications/StructuralMechanicsApplication/custom_constitutive/timoshenko_beam_elastic_constitutive_law.h

```diff
@@ -0,0 +1,202 @@
+// KRATOS  ___|  |                   |                   |
+//       \___ \  __|  __| |   |  __| __| |   |  __| _` | |
+//             | |   |    |   | (    |   |   | |   (   | |
+//       _____/ \__|_|   \__,_|\___|\__|\__,_|_|  \__,_|_| MECHANICS
+//
+//  License:         BSD License
+//                   license: StructuralMechanicsApplication/license.txt
+//
+//  Main authors:    Alejandro Cornejo
+//
+
+#pragma once
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "beam_constitutive_law.h"
+
+namespace Kratos
+{
+/**
+ * @class TimoshenkoBeamElasticConstitutiveLaw
+ * @ingroup StructuralMechanicsApplication
```

Comment:
please mention here the reference paper

Reply:
Actually I do it in the TimoshenkoElement, this is the CL which is very basic, only does strength of materials relations